packages/opam-query/opam-query.1.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "whitequark <whitequark@whitequark.org>"
+authors: "whitequark <whitequark@whitequark.org>"
+homepage: "https://github.com/whitequark/opam-query"
+bug-reports: "https://github.com/whitequark/opam-query/issues"
+license: "MIT"
+dev-repo: "git://github.com/whitequark/opam-query"
+build: [
+  "ocaml"
+  "pkg/build.ml"
+  "native=%{ocaml-native}%"
+  "native-dynlink=%{ocaml-native-dynlink}%"
+]
+depends: [
+  "ocamlfind" {build}
+  "opam-lib"
```

Comment:
Hm, actually I've just reproduced that on my own Travis CI, so it's not just an opam-repository issue...